app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ErrorsController < ApplicationController
```

Comment:
Probably need `skip_before_action :verify_authenticity_token` here as well if we're expecting this to handle json (api) requests as well?

Reply:
Done!